models/articles.models.js

```diff
@@ -15,7 +15,14 @@ exports.checkArticleIdExists = (articleId) => {
 };
 
 exports.selectArticleById = (articleId) => {
-    const queryString = `SELECT author, title, article_id, body, topic, created_at, votes,article_img_url FROM articles WHERE article_id=%L;`;
+    const queryString = `SELECT articles.author, articles.title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes, articles.article_img_url, 
+    COUNT(comments.comment_id)::int AS comment_count 
```

Comment:
Great work adding the comment account to the query, really like the ::int to ensure this is returned as a number! 🙌